### PR TITLE
STIG: add metadata

### PR DESCRIPTION
**Pull request by taroth21**
### PR creator: Description

Add metadata from 'Product Docs Metadata per deliverable' spreadsheet. 
Added a revhistory based on commit log. 

### PR creator: Are there any relevant issues/feature requests?

Link to our [internal page](https://confluence.suse.com/display/documentation/Adding+metadata+to+all+SUSE+documentation).

### CODE REVIEW

File: stig/xml/article_stig.xml [hunk 1]
```diff
@@ -35,6 +36,32 @@
...
     </abstract>
+    <meta name="title" its:translate="yes">Hardening &productname; with &stiga;</meta>
+    <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+    <meta name="description" its:translate="yes">How to audit and harden &productname; with &stig; by the &disa;</meta>
+    <meta name="social-descr" its:translate="yes">Harden &productnameshort; with &stiga;;</meta>
+    <meta name="task" its:translate="yes">
+      <phrase>Auditing</phrase>
+      <phrase>Compliance</phrase>
+    </meta>
+    <revhistory xml:id="rh-article-stig">
```
Comment (taroth21) [inline stig/xml/article_stig.xml:47]: @sounix000 : Please check if the revhistory makes sense.

**Review by janajaeger — APPROVED** [review]
LGTM. Thanks a million.

File: stig/xml/article_stig.xml (hunk 1 above)
Comment (sounix000) [reply stig/xml/article_stig.xml:47]: Yes, the revision history is correct. Must have been a painfully boring task for you to dig through the commit and merge messages. Thanks for doing it; I will keep this in mind when I do PCI-DSS and HIPPA.

**Review by sounix000 — APPROVED** [review]
LGTM.

**Pull request merged**